hw/mcu/microchip/pic32mz/src/pic32mz_periph.c

```diff
@@ -372,11 +384,23 @@ pic32mz_periph_i2c_devs(void)
     }
 }
 
+static void
+pic32mz_periph_create_eth(void)
+{
+#if MYNEWT_VAL(ETH_0)
+    int rc;
+    rc = pic32_eth_init(&eth0_cfg);
+    assert(0);
```

Comment:
this looks odd,   assert(rc == 0) maybe?

Reply:
Polishing before commit not always works as intended.
Fixed.